extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -361,6 +409,29 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     });
   }
 
+  /**
+   * Handles Fluid-related messages dispatched from SafeFrame.
+   * @param {!JsonObject} data
+   */
+  receiveMessageForFluid(data) {
+    const payload = tryParseJson(data['p']);
+    let newHeight;
+    if (!payload || !(newHeight = parseInt(payload['height'], 10))) {
+      // TODO(levitzky) Add actual error handling here.
+      this.forceCollapse();
+      return;
+    }
+    if (newHeight == parseInt(this.element.style.height, 10)) {
```

Comment:
When would this occur?  The element should have no height, correct?  Should we call onFluidResize_?  should this be getIntersectionElementLayoutBox().height?

Reply:
I suppose this is unnecessary. I just wanted to ensure we don't call attemptChangeHeight needlessly. Removed.